xcp_d/cli/parser.py

```diff
@@ -845,6 +949,59 @@ def _validate_parameters(opts, build_log, parser):
     if opts.custom_confounds:
         opts.custom_confounds = str(opts.custom_confounds.resolve())
 
+    # Define parameters based on the mode
+    if opts.exact_time == "none":
+        opts.exact_time = []
+
+    if opts.mode == "abcdbids":
+        opts.despike = False if opts.despike == "auto" else opts.despike
+        opts.cifti = True if opts.cifti == "auto" else opts.cifti
+        opts.process_surfaces = True if opts.process_surfaces == "auto" else opts.process_surfaces
+        opts.dcan_qc = True if opts.dcan_qc == "auto" else opts.dcan_qc
+        opts.exact_time = [300, 480] if opts.exact_time == "auto" else opts.exact_time
+        opts.combineruns = True if opts.combineruns == "auto" else opts.combineruns
```

Comment:
@madisoth can you weigh in on this and note any other parameters that should be set here?